### PR TITLE
permission note for copy template

### PR DESCRIPTION
**Pull request by rashminandasahoo**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by utkarshchoubey-harness — CHANGES_REQUESTED** [review]
name of permission is - core_template_copy

File: docs/platform/13_Templates/use-a-template.md [hunk 1]
```diff
@@ -64,15 +64,23 @@ You can also perform the following actions:
 Once you've made all the changes, click **Run** and then click **Run Pipeline**. The Template is deployed.
```
Comment (deepakbolangady) [inline docs/platform/13_Templates/use-a-template.md:64]: 1) Good to use a comma before "and then" in instructions. 
2) Also, good to replace "once" with "after".
3) Use lowercase in "template".

File: docs/platform/13_Templates/use-a-template.md [hunk 2]
```diff
@@ -64,15 +64,23 @@ You can also perform the following actions:
...
 
 ![](./static/use-a-template-45.png)
+
 ### Option: Copy to Pipeline
```
Comment (deepakbolangady) [inline docs/platform/13_Templates/use-a-template.md:68]: 1) "Copy to the pipeline"? (use of "the" and lowercase pipeline)
2) Global comment: per our style guide, "Begin with the most meaningful words. Do not include “Step”, “Review”, “Option” at the beginning of headings."

File: docs/platform/13_Templates/use-a-template.md [hunk 3]
```diff
@@ -64,15 +64,23 @@ You can also perform the following actions:
...
 
+
+:::note
+To copy a template, make sure you have the **core_template_copy** permission. The option to copy templates is disabled if you do not have this permission.
```
Comment (deepakbolangady) [inline docs/platform/13_Templates/use-a-template.md:72]: The second sentence in this note is implied by the first. It is not necessary. Also, should this be in a prereqs section?

File: docs/platform/13_Templates/use-a-template.md [hunk 4]
```diff
@@ -1,90 +1,99 @@
...
+This topic explains how to use an existing pipeline template in a pipeline.
 
 ### Before you begin
 
 * [Templates Overview](template.md)
-* [Create a Pipeline Template](create-pipeline-template.md)
+* [Create a pipeline template](create-pipeline-template.md)
 
-### Step: Use a Template
+### Step: Use a template
```
Comment (deepakbolangady) [inline docs/platform/13_Templates/use-a-template.md:22]: We still need to fix how we begin our titles. Again, quoting from our style guide: "Begin with the most meaningful words. Do not include “Step”, “Review”, “Option” at the beginning of headings."

Please make this change as soon as you can across your doc set. Thanks.

**Review by deepakbolangady — APPROVED** [review]
One comment was missed: from our style guide, "Begin with the most meaningful words. Do not include “Step”, “Review”, “Option” at the beginning of headings."

I'll approve for now, but please create a follow-up PR for this change. Thanks.

**Pull request merged**